network/open_channel.js

```diff
@@ -347,6 +348,31 @@ module.exports = (args, cbk) => {
           },
         );
       }],
+
+      // Set fee rate
+      setFeeRate: [
+        'candidates',
+        'openChannel',
+        ({candidates}, cbk) =>
+      {
+        // Exit early when not specifying fee rates
+        if (!args.set_fee_rate) {
+          return cbk();
+        }
+
+        return asyncEachSeries(candidates, ({public_key}, cbk) => {
+          return adjustFees({
+            cltv_delta: undefined,
+            fee_rate: String(args.set_fee_rate),
+            fs: args.fs,
+            lnd: args.lnd,
+            logger: args.logger,
+            to: [public_key],
```

Comment:
I think this will set the fee with every candidate, even failed opens

The adjust fees public key needs to be connected to the node that we actually open the channel with

Reply:
This command always opens only 1 channel, correct? Or can it do multiple?

Reply:
It only opens one, but it tries multiple nodes - if one fails it goes on to the next one

Reply:
Cool, just have to figure out to return that pubkey from the previous function I guess.

Reply:
You could also just hoist the adjustment up into that function, like after it says "opening with" then you know the final node public key